### PR TITLE
Configure service provider

### PR DESCRIPTION
**Pull request by stevenvegt**
Introduces persistence using the bbolt key value store.
Adds new apis for getting, creating and updating a service provider.
Adds new user components for managing the Service Provider.

### CODE REVIEW

**Comment by stevenvegt** [comment]
I thing this is PR is big enough, lets add the Nuts node connection and proper styling in a new PR.

File: main.go [hunk 1]
```diff
@@ -41,100 +35,31 @@ func getFileSystem(useFS bool) http.FileSystem {
...
-		log.Fatalf("unable to generate session key: %v", err)
+		log.Fatal(err)
 	}
+	defer db.Close()
 
 	e := echo.New()
 	e.HideBanner = true
 	e.Use(middleware.Logger())
-	e.Use(middleware.Recover())
+	//e.Use(middleware.Recover())
```
Comment (reinkrul) [inline main.go:50]: commented out?

File: web/src/admin/ServiceProvider.vue [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        name: '',
+        email: '',
+        phone: ''
+      }
+    }
+  },
+  created() {
+    this.fetchData()
+  },
+  methods: {
```
Comment (reinkrul) [inline web/src/admin/ServiceProvider.vue:68]: if you took the effort to create an OpenAPI spec, shouldn't we use it in the frontend as well?

**Review by reinkrul — APPROVED** [review]
looks fine. Just a demo app, after all

**Pull request merged**